packages/firestore/lib/firebase_firestore.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017, the Flutter project authors.  Please see the AUTHORS file
```

Comment:
Previous source files where copyrighted to the Chromium Authors. Unify?

Reply:
Done